NICE.Identity.Authorisation.WebAPI/Services/UsersService.cs

```diff
@@ -79,9 +79,17 @@ public User GetUser(int userId)
 			return user != null ? new User(user) : null;
 		}
 
-		public List<User> GetUsers()
+		public List<User> GetUsers(string filter = null)
 		{
-			return _context.Users.Select(user => new User(user)).ToList();
+            if (!string.IsNullOrEmpty(filter))
+            {
+                return _context.Users.Select(user => new User(user))
+                    .Where(u => (u.FirstName != null && u.FirstName.ToLower().Contains(filter.ToLower()))
+                                || (u.LastName!= null && u.LastName.ToLower().Contains(filter.ToLower()))
+                                || (u.EmailAddress!= null && u.EmailAddress.ToLower().Contains(filter.ToLower())))
```

Comment:
nice accounts also allows search by the nameidentifier. could this be added here please?

Reply:
Done